jobQueue/setup.js

```diff
@@ -40,6 +40,7 @@ if (featureToggles.isFeatureEnabled('instrumentKue')) {
 module.exports = {
   jobConcurrency: {
     BULKUPSERTSAMPLES: conf.bulkUpsertSampleJobConcurrency,
+    GET_HIERARCHY: 1,
```

Comment:
Would you mind setting up a conf.getHierarchyJobConcurrency variable like the one for bulk upsert? It would be nice to be able to tune this from env. variables if needed.